driver/modern_bpf/definitions/struct_flavors.h

```diff
@@ -58,4 +58,75 @@ struct modern_bpf__kernel_timex_timeval
 	long long int tv_usec;
 };
 
+struct inode___v6_6 {
```

Comment:
This shuld go into the 

```
#ifndef BPF_NO_PRESERVE_ACCESS_INDEX
#pragma clang attribute push(__attribute__((preserve_access_index)), apply_to = record)
#endif
```

used above, otherwise, no relocation will happen.
Moreover it is not needed to report the entire struct just the `struct timespec64 __i_ctime;` field should be enough

Reply:
> Moreover it is not needed to report the entire struct just the struct timespec64 __i_ctime; field should be enough

how can it know where to map the new field (ie: at which offset)? Magic?

Reply:
I see now! Thanks!

Reply:
libbpf magic, see here https://nakryiko.com/posts/bpf-core-reference-guide/#handling-incompatible-field-and-type-changes